### PR TITLE
fix typo in docs, update copy on id verification modal

### PR DESCRIPTION
**Pull request by eastandwestwind**
Closes https://github.com/ethyca/fides/issues/1898

### Code Changes

* [ ] fix typo in docs
* [ ] update copy on id verification modal

### Steps to Confirm

* [ ] In `fides.toml`, set `subject_identity_verification_required = true`, and add the following:
```
[notifications]
notification_service_type = "mailgun"
```
* [ ] Run `nox -s dev`
* [ ] Open new terminal, `cd` into privacy center, then run `npm run dev`
* [ ] Nav to `http://localhost:3000`
* [ ] Submit new privacy request or consent request, using email identity
* [ ] Confirm modal text is generic, and doesn't reference email or phone number:
Screen Shot 2022-12-14 at 11 30 10 AM

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation Updated:
  * [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  * [ ] documentation issue created (tag docs-team to complete issue separately)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

File: src/fides/api/ctl/migrations/versions/1f61c765cd1c_merge_alembic_heads.py [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 
 # revision identifiers, used by Alembic.
-revision = '1f61c765cd1c'
-down_revision = ('8f84fad4e00b', 'b72541d79f10')
+revision = "1f61c765cd1c"
+down_revision = ("8f84fad4e00b", "b72541d79f10")
```
Comment (eastandwestwind) [inline src/fides/api/ctl/migrations/versions/1f61c765cd1c_merge_alembic_heads.py:14]: `nox -s black` formatter was complaining about this already committed code

File: tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py [hunk 2]
```diff
@@ -4519,7 +4519,10 @@ def test_create_privacy_request_notification_info_deletes_addresses(
     ):
         PrivacyRequestNotifications.create(
             db=db,
-            data={"email": "test@email.com, test2@email.com", "notify_after_failures": 10},
+            data={
+                "email": "test@email.com, test2@email.com",
+                "notify_after_failures": 10,
+            },
```
Comment (eastandwestwind) [inline tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py:4525]: same as above

File: src/fides/api/ctl/migrations/versions/1f61c765cd1c_merge_alembic_heads.py (hunk 1 above)
Comment (sanders41) [reply src/fides/api/ctl/migrations/versions/1f61c765cd1c_merge_alembic_heads.py:14]: I have seen this happening lately with black, mypy, and pylint. Not sure why CI isn't catching them the first time (or in some cases several times) through.

**Review by sanders41 — APPROVED** [review]
lgtm

**Pull request merged**